chromadb/api/models/CollectionCommon.py

```diff
@@ -510,35 +461,27 @@ def _process_upsert_or_update_request(
         )
 
         normalized_embeddings = (
-            self._normalize_embeddings(unpacked_embedding_set["embeddings"])
-            if unpacked_embedding_set["embeddings"] is not None
+            self._normalize_embeddings(unpacked_record_set["embeddings"])
+            if unpacked_record_set["embeddings"] is not None
```

Comment:
I think we can fix up the logic here to be cleaner, and simplify `compute_embeddings` into one function.

Something like:

```python
if unpacked_record_set["embeddings"] is not None:
    unpacked_record_set["embeddings"] = self._normalize_embeddings(unpacked_record_set["embeddings"])
else:
    unpacked_record_set["embeddings"] = self._compute_embeddings_upsert_or_update_request(
    embeddings=normalized_embeddings,
    documents=unpacked_record_set["documents"],
    images=unpacked_record_set["images"],
)

```

Reply:
put embeddings normalization inside unpacking